data/dogs/Custom.css

```diff
@@ -0,0 +1,135 @@
+body { 
+background-image: url('https://i.imgur.com/o6baoVB.jpg');
```

Comment:
If this can be used it should be moved to Autumn; however, are there any copyright issues?

Reply:
No, it's my own picture of my own dog ;)

Reply:
Btw what's autumn @Rexogamer

Reply:
By that I mean "reupload it to Revolt and copy the image link from there"

Reply:
Oh okay. How do I do that?

Reply:
@Rexogamer Hey sorry I forgot about this. Will fix today

Reply:
Just following up on this if you're still interested in having your theme included.

Reply:
Hmmm, sure. Will need to make the modifications though, which I completely forgot about haha

I'll fix it in 7~isch hours?